ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTimeStampLabel.qml

```diff
@@ -5,17 +5,25 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 
 StatusBaseText {
-    property alias tooltip: tooltip
+    id: root
+    property double timestamp: Date.now()
 
     color: Theme.palette.baseColor1
     font.pixelSize: 10
     visible: !!text
+    text: Qt.formatTime(new Date(timestamp), Qt.locale(), Locale.ShortFormat)
     StatusToolTip {
         id: tooltip
         visible: hhandler.hovered && !!text
         maxWidth: 350
     }
     HoverHandler {
         id: hhandler
+        onHoveredChanged: {
+            if(hhandler.hovered) {
+                const dateTime = Qt.formatDateTime(new Date(root.timestamp), Qt.locale(), Locale.LongFormat)
+                if(dateTime) tooltip.text = dateTime
```

Comment:
How has this tooltip worked before? From what I see, there was no tooltip.text assignment/binding at all :thinking:

Reply:
The tooltip itself was public and the text was assigned from the outside as tooltip.text